pilot/helpers/Debugger.py

```diff
@@ -47,7 +47,7 @@ def debug(self, convo, command=None, user_input=None, issue_description=None, is
 
             if ask_before_debug or i > 0:
                 print('yes/no', type='button')
-                answer = ask_user(self.agent.project, 'Can I start debugging this issue?', require_some_input=False)
+                answer = ask_user(self.agent.project, 'Can I start debugging this issue [Y/n/error details]?', require_some_input=False)
```

Comment:
I would put same capitalisation:
`Y/N/Error details`
or
`y/n/error details`

Reply:
Reason for this capitalisation is that Y is default